<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hall</class>
 <widget class="QWidget" name="Hall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidgetHall">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QWidget" name="page1player">
    <property name="styleSheet">
     <string notr="true">#page1player {
	background-color: grey;
}</string>
    </property>
    <widget class="QPushButton" name="buttonStartGame">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>480</y>
       <width>230</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#buttonStartGame {
	background-color: black;
	font-size: 30px;
}</string>
     </property>
     <property name="text">
      <string>Start game</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWaiting">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>490</y>
       <width>561</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Esperando a que el host inicie la partida...</string>
     </property>
    </widget>
    <widget class="QFrame" name="containerOnlyOnePlayer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>391</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="labelOnlyOnePlayer">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Player</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelmageDuck">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>160</width>
        <height>160</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/DuckSelection.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="containerTwoPlayers">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>651</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="labelPlayer_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Player 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlayer_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Player 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelImageDuckP1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>160</width>
        <height>160</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/DuckSelection.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelImageDuckP2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>160</width>
        <height>160</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/DuckSelection.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
